lib/lita/handlers/animate.rb

```diff
@@ -44,6 +52,27 @@ def safe_value
         safe = "active" unless ["active", "moderate", "off"].include?(safe)
         safe
       end
+
+      def valid_extension?(result)
+        if /^.*\.(jpg|JPG|png|PNG|jpeg|JPEG)/.match(result)
```

Comment:
I copy pasted this from SO. I'm a bad person and didn't think about what I was doing.